src/node/certs.h

```diff
@@ -1,12 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the Apache 2.0 License.
 #pragma once
+
 #include "entities.h"
 
 namespace ccf
 {
-  using CertDERs = kv::Map<Cert, ObjectId>;
+  using CertDERs = kv::Map<Cert, CallerId>;
```

Comment:
I think this table can also be removed, since we can also recompute this `Cert -> CallerId` when needed?

Reply:
Ah! Very good point.

Reply:
Done now. Let's see if hashing the DER certificate on every request has any impact on performance.

Reply:
In the unlikely event that it does, we can always memoise it like we do for verifiers.

Reply:
As per https://github.com/microsoft/CCF/pull/2279#issuecomment-792863287, these changes don't impact performance 👍